api-reference/headless-portal.mdx

```diff
@@ -22,41 +61,38 @@ To display a list of support requests you will need to fetch the appropriate thr
   });
 ```
 
-_Tenants allow you to structure your customers in Plain in the same way as they are structured in your product. For example if within your product customers are organised in a ‘team’ then you would create one tenant per team in your product._
+If in Step 1 you opted for not using Tenants then you can also filter by customerId here in order to fetch just the threads belonging to a specific customer.
 
-<Note>
-  When filtering by tenants, if you want to display threads which were created by incoming emails or
-  Slack messages you will need to set the correct tenant id on those threads. This is best done by
-  using the [Thread Created Webhook](/api-reference/webhooks/thread-created).
-</Note>
+This will give you back a list of threads but not give you back the content of a thread. To do so you must use this query. This will fetch all relevant message timeline entries. This example is showing how to do this via GraphQL directly but this can also be done with our SDK using the `rawRequest` method.
 
-Alternatively you can filter threads by [company](/api-reference/graphql/companies) if you don't need to break access down by tenant.
+<Snippet file="graphql/get-thread-timeline.mdx" />
+</Step>
 
-```Javascript
-  const threads = await plainClient.getThreads({
-    filters: {
-      companyIdentifiers: [{ companyId: customerCompanyId }],
-      // You can also query company by its domain name
-      // companyIdentifiers: [{ companyDomainName }],
-      statuses: [ThreadStatus.Todo, ThreadStatus.Snoozed],
-    },
-  });
-```
+<Step title="Allow customers to submit new threads">
+To submit new support requests from your portal you can create a contact form which, when submitted, create a thread within Plain.
 
-## Authenticating
+What is important is that when you create the thread you pre-fill the customer's tenant id. This ensures that the support request they create is then visible in the support portal.
 
-In order to pass in the `tenantExternalId` or `customerCompanyId` shown above you will need to authenticate the user of the support portal. This is not shown in the example repository since it will be different for every application.
+You can read more on [how to create a thread here](/api-reference/graphql/threads/create).
 
-Once you have authenticated you may be able to get the id of their tenant directly from the claims, or you may need to query your backend systems.
+One thing to not is that you are not constrained at all here by the data you want to gather. Although a simple form with just a message and a title does work, it's typically much more useful and powerful to ask structured questions such as what the support request is about, how urgent it is and much more.
```

Comment:
```suggestion
One thing to note is that you are not constrained at all here by the data you want to gather. Although a simple form with just a message and a title does work, it's typically much more useful and powerful to ask structured questions such as what the support request is about, how urgent it is and much more.
```

`much more` could link to a page where we describe e.g. labels, priorities, etc?